### PR TITLE
Disable sender config option

### PR DESCRIPTION
**Pull request by brandonjoyce**
We're attempting to use the Plug provider for ExSentry, and we need a way to disable it in our dev environments.  This is one way to do it, but I'm open to other possible solutions.  What do you think?

Collaborating w/ @ringmaster

### CODE REVIEW

File: lib/exsentry.ex [hunk 1]
```diff
@@ -26,7 +26,15 @@ defmodule ExSentry do
   3. Optional: To configure the default ExSentry client, specify your
      Sentry DSN in `config.exs`:
 
-          config :exsentry, dsn: "your-dsn-here"
+          config :exsentry,
+            dsn: "your-dsn-here",
+            sender_opts: [ # this section optional
+              disabled: false, # stops sending to sentry
+              delay: 1000, # delay between retries
```
Comment (gamache) [inline lib/exsentry.ex]: FWIW, when submitting a PR, it's not good practice to include docs for features that are not yet implemented.

**Comment by gamache** [comment]
The Sentry-approved way of disabling the client is to pass in the empty string as DSN.  This is documented on their client API page: https://docs.getsentry.com/hosted/clientdev/

ExSentry supports this feature already, so I am going to close this PR.  I should make this clear in the ExSentry documentation though.  Thanks for pointing this out!

File: lib/exsentry.ex (hunk 1 above)
Comment (brandonjoyce) [inline lib/exsentry.ex]: I think those ARE implemented.  Am I wrong?

**Comment by brandonjoyce** [comment]
Well, that's an easy way to do it!  We'll give it a try.

**Comment by brandonjoyce** [comment]
Looks like that will work.  Thanks for your work on this BTW!

**Pull request closed without merging**